AWSConnect/AWSConnectModel.h

```diff
@@ -1799,6 +1846,42 @@ typedef NS_ENUM(NSInteger, AWSConnectUnit) {
  */
 @property (nonatomic, strong) NSString * _Nullable participantToken;
 
+@end
+
+/**
+ 
+ */
+@interface AWSConnectStartContactRecordingRequest : AWSRequest
+
+
+/**
+ <p>The identifier of the contact.</p>
+ */
+@property (nonatomic, strong) NSString * _Nullable contactId;
+
+/**
+ <p>The identifier of the contact. This is the identifier of the contact associated with the first interaction with the contact center.</p>
+ */
+@property (nonatomic, strong) NSString * _Nullable initialContactId;
+
+/**
+ <p>The identifier of the Amazon Connect instance.</p>
+ */
+@property (nonatomic, strong) NSString * _Nullable instanceId;
+
+/**
+ <p>Who is being recorded.</p>
+ */
+@property (nonatomic, strong) AWSConnectVoiceRecordingConfiguration * _Nullable voiceRecordingConfiguration;
+
+@end
+
+/**
+ 
+ */
+@interface AWSConnectStartContactRecordingResponse : AWSModel
+
```

Comment:
Why is this empty?

Reply:
Per [the docs](https://docs.aws.amazon.com/connect/latest/APIReference/API_StartContactRecording.html):

> If the action is successful, the service sends back an HTTP 200 response with an empty HTTP body.

So it's expected that this is empty